packages/html/ds/src/textarea/textarea.ts

```diff
@@ -0,0 +1,57 @@
+import {
+  BaseComponent,
+  BaseComponentOptions,
+  initialiseModule,
+} from '../common/component';
+
+export type TextareaProps = BaseComponentOptions;
+
+export class Textarea extends BaseComponent<TextareaProps> {
+  calculateRemainingChars: () => void;
+  getAllTextarea: NodeListOf<HTMLTextAreaElement>;
+
+  constructor(options: TextareaProps) {
+    super(options);
+    this.getAllTextarea = document.querySelectorAll('textarea');
+
+    this.calculateRemainingChars = () => {
+      try {
```

Comment:
Put it inside the for-loop.
In this way, if you have two textarea, the first without the counter and the second with the counter, it will quit too soon

Reply:
good catch! fixed.